usagov/vote-gov
# VOTE-277 - Cypress Axe Plugin

Link to ticket -> https://bixal-projects.atlassian.net/browse/VOTE-277

This Pr is the adding and setting up of the Cypress Axe Plugin.  

Additions:
- added Cypress-axe plug-in and axe core to the project 
- added the `axe.spec.js` file to run on the pages there were identified in the testing plan 
- added an accessibility testing folder and currently hold the axe test and any other accessibility tests in the future 
- added a new testing script of `npm run cy:axe` which will run that axe test in the terminal and report any issues in the terminal 

How to test:
There are 2 ways to test this addition 
- pull branch on your local
- run `npm run start`
- spilt terminal 
* First option -> run `npm run cy:axe` this will run the test headless and will add any errors to a table in the terminal 
* Second option -> run `npm run cy:open` this will open the cypress runner and can select to run the test `axe.spec.js` this will allow you to see the test run and will still add any errors in a table in the terminal but will also highlight the issue in the browser

link to documentation for plug-in -> https://github.com/component-driven/cypress-axe
